maximize-tensorflow-performance-on-cpu-considerations-and-recommendations-for-inference/maximize-tensorflow-performance-on-cpu-considerations-and-recommendations-for-inference.md

```diff
@@ -0,0 +1,287 @@
+To fully utilize the power of Intel® architecture (IA) for high performance, you can enable TensorFlow* to be powered by Intel’s highly optimized math routines in the Intel® oneAPI Deep Neural Network Library (oneDNN). oneDNN includes convolution, normalization, activation, inner product, and other primitives.
+
+The oneAPI Deep Neural Network Library (oneDNN) optimizations are now available both in the official x86-64 TensorFlow and  Intel® Optimization for TensorFlow* after v2.5. Users can enable those CPU optimizations by setting the the environment variable **TF_ENABLE_ONEDNN_OPTS=1** for the official x86-64 TensorFlow after v2.5.
+
+Most of the recommendations work on both official x86-64 TensorFlow and  Intel® Optimization for TensorFlow. Some recommendations such as OpenMP tuning only applies to Intel® Optimization for TensorFlow.
+
+For setting up Intel® Optimization for TensorFlow* framework, please refer to this [installation guide](https://www.intel.com/content/www/us/en/developer/articles/guide/optimization-for-tensorflow-installation-guide.html).
+
+## Maximum Throughput vs. Real-time Inference
+You can perform deep learning inference using two different strategies, each with different performance measurements and recommendations. The first is Max Throughput (MxT), which aims to process as many images per second as possible, passing in batches of size > 1. For Max Throughput, you achieve better performance by exercising all the physical cores on a socket. With this strategy, you simply load up the CPU with as much work as you can and process as many images as you can in a parallel and vectorized fashion.
+
+An altogether different strategy is Real-time Inference (RTI) where you typically process a single image as fast as possible. Here you aim to avoid penalties from excessive thread launching and orchestration among concurrent processes. The strategy is to confine and execute quickly. The best-known methods (BKMs) differ for these two strategies.
+
+## TensorFlow Graph Options Improving Performance
+Optimizing graphs help improve latency and throughput time by transforming graph nodes to have only inference related nodes and by removing all training nodes.
+
+Users can use tools from TensorFlow github.  
+
```

Comment:
why do we remove INC session?

However, INC indeed remove below docs from v2.0.
https://github.com/intel/neural-compressor/blob/v1.14.2/docs/graph_optimization.md

Reply:
So do you prefer keeping and link to v1.14.2 doc?

Reply:
possible to understand why it is removed from INC Team?
let us keep your current contents because it has been removed.